rancher/v1.2/en/quick-start-guide/index.md

```diff
@@ -79,7 +79,7 @@ If you click on the dropdown of the **_first-container_**, you will be able to p
 Rancher will display any containers on the host even if the container is created outside of the UI. Create a container in the host's shell terminal.
 
 ```bash
-$ docker run -d -it --name=second-container ubuntu:14.04.2
+$ docker run -it --name=second-container ubuntu:14.04.2
```

Comment:
You need the `-d` or the container will stop once you get back to the command line.